yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -48,7 +71,7 @@ export class Archiver implements L2BlockSource, UnverifiedDataSource {
    * @param config - The archiver's desired configuration.
    * @returns - An instance of the archiver.
    */
-  public static async createAndSync(config: ArchiverConfig) {
+  public static createAndSync(config: ArchiverConfig) {
```

Comment:
I feel like this function should still offer the ability to await until the initial sync has completed. Perhaps an argument to start() that determines it calls and awaits sync() before kicking off the running promise.

Reply:
Addressed in 8a863ccfbc8fcad01975a121cfa6c529ac56a59b